content/en/docs/usage/certificate.md

```diff
@@ -75,6 +85,9 @@ The signed certificate will be stored in a `Secret` resource named
 `example-com-tls` in the same namespace as the `Certificate` once the issuer has
 successfully issued the requested certificate.
 
+If `secretTemplate` is present, annotations and labels set in this property
+will be copied over to `example-com-tls` secret. Both properties are optional.
```

Comment:
Just wondering, how do we let people know that this is a feature that only exists since 1.5? The user would have to look for differences between 1.4 and 1.5 to find that out.

I'd go with smth like:

```suggestion
If `secretTemplate` _(since cert-manager 1.5)_ is present, annotations and labels set in this property
will be copied over to `example-com-tls` secret. Both properties are optional.
```

@irbekrm what do you think? 😅

Reply:
Our docs are versioned so I don't think this is too much of a concern.

Reply:
@JoshVanL It is very frustrating to have to select one by one the documentation versions to find when a feature was introduced and know if your version of cert-manager works with that feature.

I am often in the situation where I start using a feature and realize later on that the feature was introduced in a later version.

Reply:
I guess we may want to merge this into release-next branch, not into master though?

Reply:
updated and rebased @irbekrm

Reply:
I've just added a release note based on the PR description and code comments, let me know if it needs to be rewritten. @benlangfeld I took the liberty to add your name to the contributors' list above.

Reply:
Looks like our stupid spellchecker doesn't like the github usernames. You'll have to add them to the .spelling file. See https://github.com/cert-manager/website/blame/master/.spelling#L265

Reply:
Usernames added to .spelling.

Reply:
@jonathansp I'm flattered, but I didn't do anything nearly worth being included in a list of contributors :)